packages/aws-rfdk/lib/core/lib/exporting-log-group.ts

```diff
@@ -111,10 +113,25 @@ export class ExportingLogGroup extends Construct {
       threshold: 1,
     });
 
+    // Define log retention retry options to reduce the risk of the rate exceed error
+    // as the default create log group TPS is only 5. Make sure to set the timeout of log retention function
+    // to be greater than total retry time. That's because if the function that is used for a custom resource
+    // doesn't exit properly, it'd end up in retries and may take cloud formation an hour to realize that
+    // the custom resource failed.
     const logRetention = new LogRetention(this, 'LogRetention', {
       logGroupName: props.logGroupName,
       retention: retentionInDays,
+      logRetentionRetryOptions: {
+        base: Duration.millis(200),
+        maxRetries: 7,
```

Comment:
Checking the math... 

I think that the backoff code in the JS SDK is [here](https://github.com/aws/aws-sdk-js/blob/aa6049093165b84eac975aa50c3a44e23cbfe4af/lib/util.js#L866) ( called from [here](https://github.com/aws/aws-sdk-js/blob/aa6049093165b84eac975aa50c3a44e23cbfe4af/lib/service.js#L551) amongst other places ).

Looks like the total amount of delay time over all retries will be `(2^0 + 2^1 + ... + 2^(maxRetries-1} ) * 200ms`. With maxRetries of 6 that puts our max delay at 25.2s. So, we should be good with these settings and increasing the Lambda Function's timeout to 30s.